### DISCUSSION TITLE
Not Working Issue

### DISCUSSION THREAD

**Discussion by ruruyaa** [amvstrm/amvstrm#39] [Discussions]
buttons Are Not Working And Stucked On This Interface
![IMG_20240227_161935](https://github.com/amvstrm/amvstrm/assets/101450382/1a0c64f4-4049-4559-854c-128c83f53018)

And This Is My Terminal Tab Screenshot
![IMG_20240227_161818](https://github.com/amvstrm/amvstrm/assets/101450382/969b5353-6655-47f2-a934-2890cca5043b)

Please Reply On This Discussion Page as soon as possible

TO, Mod And Admin Of amvstrm

**Comment by NYT92** [comment]
I have already created my own github codespaces and the problem did not happen to me. Can you show me your browser console and network?

**Reply by ruruyaa** [reply]
Browser Console And Tab 
can U give me a Yours plz

**Reply by NYT92** [reply]
I'm not in the codespaces right now but it perfectly fine for me after starting the project. When I checked the network tab and console, there is no error occured or any failed requests.

Those error that I see can be cause by network issues.

Can you recheck again now?

**Reply by ruruyaa** [reply]
what Is Network Tab On VS Code 
Here's My Scrrenshot
![Screenshot_2024-03-05-09-00-31-26_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/amvstrm/amvstrm/assets/101450382/2ca83558-6b53-4021-af7a-9a8b6134b00f)

I am Replit User So , I Not Know What it is 😅😅

**Reply by NYT92** [reply]
That might be hard to debug, didn't know you use the phone too...

I still don't understand why you got those error even though I try to replicate it fine

Have you tried to start a new codespaces with the project again?
